modules/runner/src/testFixtures/java/org/apache/ignite/internal/ClusterPerClassIntegrationTest.java

```diff
@@ -74,6 +74,10 @@ public abstract class ClusterPerClassIntegrationTest extends IgniteIntegrationTe
             + "      netClusterNodes: [ {} ]\n"
             + "    }\n"
             + "  },\n"
+            + "  storages: {\n"
```

Comment:
I would propose to use the following formatting:

```
+ "  storages.profiles: {"
+ "        default_aipersist.engine: aipersist, "
+ "        default_aimem.engine: aimem, "
+ "        default_rocksdb.engine: rocksDb"
+ "  },\n"
```

Reply:
Fixed